src/domain/mainwindow.py

```diff
@@ -65,6 +65,7 @@ def __init__(self):
             os.getcwd(), "src", "img","mainwindow_icon.jpg")))
 
         # Update mainwindow UI methods
+        self._init_logging_dropdown()
         self.update_toolbar_status()
         logger.info('Welcome to WADAS!')
```

Comment:
A unique, uniform, coding convention should be applied for string literals: either you use the single quote or the double quote in all code (exceptions: when there are quotes or double quotes inside the strings, then you use the opposite for escaping them).

Reply:
I suggest you to use Black for applying the official Python formatting (as defined on PEP8): https://black.readthedocs.io/en/stable/integrations/editors.html